backend/dataall/modules/metadata_forms/services/metadata_form_service.py

```diff
@@ -295,25 +298,75 @@ def get_mf_permissions(entityUri):
     @MetadataFormAccessService.can_perform(UPDATE_METADATA_FORM_FIELD)
     def create_metadata_form_version(uri, copyVersion):
         with get_context().db_engine.scoped_session() as session:
+            mf = MetadataFormService.get_metadata_form_by_uri(uri)
             new_version = MetadataFormRepository.create_metadata_form_version_next(session, uri)
             if copyVersion:
                 mf_fields = MetadataFormRepository.get_metadata_form_fields(session, uri, copyVersion)
                 for field in mf_fields:
                     new_field = MetadataFormRepository.create_metadata_form_field(
                         session, uri, field.__dict__, new_version.version
                     )
+
+            all_attached = MetadataFormRepository.get_all_attached_metadata_forms(session, uri)
+            for attached in all_attached:
+                owner = MetadataFormService.get_entity_owner(attached)
+                if owner:
+                    NotificationRepository.create_notification(
+                        session,
+                        recipient=owner,
+                        target_uri=f'{attached.entityUri}|{attached.entityType}',
+                        message=f'New version {new_version.version} is available for metadata form "{mf.name}"',
```

Comment:
can we add to message "`... for entity {attached.entityUri}|{attached.entityType}`" 

this way if I have 1 form attached to 5 datasets and 5 envs - I can make sense to why I have received 10 notifications and they are all not the same message

Reply:
done